ds/ds.php

```diff
@@ -1272,21 +1326,126 @@ public function copy(): Collection
         {
         }
 
+        /**
+         * Returns the result of removing all keys from the current instance that are present in a given map.
+         *
+         * A \ B = {x ∈ A | x ∉ B}
+         *
+         * @param Map $map The map containing the keys to exclude in the resulting map.
+         *
+         * @return Map The result of removing all keys from the current instance that are present in a given map.
+         *
+         * @link https://www.php.net/manual/en/ds-map.diff.php
+         */
+        public function diff(Map $map): Map
+        {
+        }
+
+        /**
+         * Creates a new map using a callable to determine which pairs to include
+         *
+         * @param callable $callback Optional callable which returns TRUE if the pair should be included, FALSE
+         * otherwise. If a callback is not provided, only values which are TRUE (see converting to boolean) will be included.
+         *
+         * @return Map
+         *
+         * @link https://www.php.net/manual/en/ds-map.filter.php
+         */
+        public function filter(callable $callback): Map
+        {
+        }
+
+        /**
+         * Returns the first pair in the map
+         *
+         * @return Pair The first pair in the map.
+         *
+         * @throws UnderflowException if empty
+         */
+        public function first(): Pair
+        {
+        }
+
+        /**
+         * Returns the value for a given key, or an optional default value if the key could not be found.
+         * <p>
+         * <b>Note:</b> Keys of type object are supported. If an object implements Ds\Hashable, equality will be
+         * determined by the object's equals function. If an object does not implement Ds\Hashable, objects must be references to the same instance to be considered equal.
+         * <p>
+         * <b>Note:</b> You can also use array syntax to access values by key, eg. $map["key"].
+         * <p>
+         * <b>Caution:</b> Be careful when using array syntax. Scalar keys will be coerced to integers by the engine. For
+         * example, $map["1"] will attempt to access int(1), while $map->get("1") will correctly look up the string key.
+         *
+         * @param mixed $key The key to look up.
+         * @param mixed $default The optional default value, returned if the key could not be found.
+         *
+         * @return The value mapped to the given key, or the default value if provided and the key could not be found in the map.
+         *
+         * @throws OutOfBoundsException if the key could not be found and a default value was not provided.
+         */
+        public function get($key, $default)
```

Comment:
please add `null` as default value for `$default` as this parameter is optional

Reply:
I did a bit more than that and re-checked the whole file for similar mistakes.